community/values/src/main/java/org/neo4j/values/storable/PointValue.java

```diff
@@ -107,12 +107,14 @@ public boolean eq( Object other )
     }
 
     @Override
-    public int compareTo( Value otherValue )
+    public int compareTo( PointValue other )
+    {
+        return unsafeCompareTo(other);
```

Comment:
Same as above, we can invert who calls who between `compareTo` and `unsafeCompareTo`